.pre-commit-config.yaml

```diff
@@ -37,6 +37,24 @@ repos:
         args: ["--add-ignore=D107,D105"]
         additional_dependencies:
           - toml
+  - repo: local
+    hooks:
+    - id: fmt-eko
+      name: fmt-eko
+      description: Format eko files with cargo fmt.
+      entry: cargo fmt --manifest-path crates/eko/Cargo.toml --
```

Comment:
Unfortunately, this has a drawback: there is no `cargo` installed on the `pre-commit.ci`

Reply:
that is correct - however, I don't believe that we can ever run rust there, so the best we can do is to skip this in the CI and rely on the developers to run ... (note that I copied this from an existing hook, so they do nothing more - but in fact less that's why I needed to copy in the first place)

Reply:
How would you skip in the CI in practice?

Reply:
https://github.com/NNPDF/eko/pull/189/commits/02b772f0979390a1aa048da89bf5f8ba840334bd